insights/parsr/__init__.py

```diff
@@ -1240,3 +1240,4 @@ def skip_none(x):
 SingleQuotedString = Char("'") >> String(set(string.printable) - set("'"), "'") << Char("'")
 DoubleQuotedString = Char('"') >> String(set(string.printable) - set('"'), '"') << Char('"')
 QuotedString = Wrapper(DoubleQuotedString | SingleQuotedString) % "quoted string"
+EmptyAttr = String('"\'', min_length=2)
```

Comment:
@huali027 this seems more specific to the parser you are modifying than something that should be included in `parsr`.  The category of "empty attributes" could cover a lot of different cases.

Reply:
@bfahr, besides httpd, nginx is the same. I thought maybe the others have the same configuration to set empty string, so I put it in the `parsr`. Do you suggest to put it to httpd_conf.py first?

Reply:
@huali027 yes, I was suggesting that it be kept in httpd_conf.py instead.  It's still pretty specific even if its both Apache and Nginx config files.  I think that "EmptyAttr" could be interpreted much more broadly and in different ways which would lead to confusion.

Reply:
@bfahr, sure, I'll update it to httpd_conf.py.